### ISSUE TITLE
Spider: Wayne County Land Bank

### ISSUE THREAD

**Issue by pjsier** [City-Bureau/city-scrapers#851]
URL: https://public-wclb.epropertyplus.com/landmgmtpub/app/base/customPage
Spider Name: `wayne_land_bank`
Agency Name: Wayne County Land Bank

See the [contribution guide](https://cityscrapers.org/docs/development/#contribute) for information on how to get started

This one is interesting because you'll need to start with a POST request instead of a GET request. In this case, you'll need to override the spider's `start_requests` method with something like this:

```python
def start_requests(self):
    yield scrapy.FormRequest(self.start_urls[0], callback=self.parse, dont_filter=True)
```

**Comment by radoslawkrolikowski** [comment]
Hello. I would like to work on that spider.

**Comment by pjsier** [comment]
@radoslawkrolikowski sounds good!

**Comment by radoslawkrolikowski** [comment]
I encountered an issue with `city_scrapers_core.utils.file_response`. The copy of the web page that was created automatically wasn't the proper response, so I had to replace it with response.body html. Now the` file_response` is doubling each backslash in the `test_response`:
- ['\\\r\\\n\\t\\\t<div class="col-lg-4..."
Instead of:
- ['\r\n\t\t<div class="col-lg-4..."
thus, my regular expressions work while scraping, but they don't match anything at testing time.
Any advices, how to prevent such behavior, would be very appreciated.

**Comment by pjsier** [comment]
Hmm, not sure I'm following specifically where the issue started, but instead of using `response.body` it might be worth trying to save the page with the correct response in the browser as HTML. Let me know if that seems to fix it

**Comment by radoslawkrolikowski** [comment]
Saving response in the browser as HTML helped, now everything is working correctly. Thanks a lot.

**Comment by pjsier** [comment]
Closed by #885, thanks @radoslawkrolikowski!
